shared/aux.go

```diff
@@ -72,9 +73,9 @@ func RunCommandOnNode(cmd, ip string) (string, error) {
 		!strings.Contains(cleanedStderr, "2")) {
 		return cleanedStderr, nil
 	} else if cleanedStderr != "" {
-		return "", fmt.Errorf("command: %s failed with error: %v\n", cmd, stderr)
+		return "", fmt.Errorf("command: %s failed with error: %v", cmd, stderr)
```

Comment:
please roolback the \n, makes huge difference on console

Reply:
done